src/Combinatorics/SimplicialComplexes.jl

```diff
@@ -637,7 +637,7 @@ function deletion(K::SimplicialComplex, face::Union{<:AbstractSet{Int},<:Abstrac
 end
 
 @doc raw"""
-    automorphism_group(K::SimplicialComplex; action=:on_vertices)
+    automorphism_group(K::SimplicialComplex)
```

Comment:
Change seems unrelated. Maybe a faulty merge?

Reply:
yes this change is constantly coming back

Reply:
it's haunting me